### DISCUSSION TITLE
"export 'default' (imported as 'Index') was not found in './pages/index/index'

### DISCUSSION THREAD

**Discussion by ShawnZeng** [NervJS/taro#11122] [Q & A]
Index文件里面明明有export default，却偏偏说我没有。百思不得其解。是不是Bug？
![image](https://user-images.githubusercontent.com/29124100/149767232-44f59361-49c8-488e-9c58-15bab6d71bf2.png)

**Comment by ZakaryCode** [comment]
编译平台是？
